src/components/Shows/index.js

```diff
@@ -0,0 +1,54 @@
+import { compose } from 'redux'
+import { connect } from 'react-redux'
+import { firebaseConnect } from 'react-redux-firebase'
+import Shows from './Shows'
+import { values, mapObjIndexed, reverse } from 'ramda'
+import moment from 'moment'
+
+const setMomentDate = show => {
+  show._date = moment(show.date).format('dddd, MMMM Do')
+  return show
+}
+
+const sortByDate = shows => shows.sort((a, b) => {
+  a = new Date(a.date)
+  b = new Date(b.date)
+  return a > b ? -1 : a < b ? 1 : 0
+})
```

Comment:
I love `comparator` (and `R.sort`) for situations like this: http://ramdajs.com/docs/#comparator

could do something like
```es6
const compareDate = comparator((a, b) => new Date(a.date) < new Date(b.date));
const sortByDate = sort(compareDate);
```

Reply:
Very clean. Ramda comes through again